libpromises/mod_files.c

```diff
@@ -227,6 +227,15 @@ static const ConstraintSyntax delete_constraints[] =
 
 static const BodySyntax delete_body = BodySyntaxNew("delete", delete_constraints, NULL, SYNTAX_STATUS_NORMAL);
 
+static const ConstraintSyntax fsattrs_constraints[] =
+{
+    CONSTRAINT_SYNTAX_GLOBAL,
+    ConstraintSyntaxNewBool("immutable", "true/false whether or not to immutable flag should be set", SYNTAX_STATUS_NORMAL),
```

Comment:
s/or not to/or not the/

or

"true/false whether or not to set the immutable flag"